app/inputs/tempus_dominus_date_input.rb

```diff
@@ -0,0 +1,36 @@
+class TempusDominusDateInput < SimpleForm::Inputs::Base
+  def input(_wrapper_options)
+    template.content_tag(:div, class: 'input-group date tempus-dominus-date', data: { target_input: 'nearest' }, id: "#{object_name}_#{attribute_name}") do
+      template.concat @builder.text_field(attribute_name, input_html_options)
+      template.concat div_button
+    end
+  end
+
+  def input_html_options
+    super.merge(class: 'form-control datetimepicker-input', value: input_value, data: { target: "##{object_name}_#{attribute_name}" })
+  end
+
+  def input_value
+    @builder.object.send(attribute_name).try(:strftime, "%d/%m/%Y") || Time.zone.today.strftime("%d/%m/%Y")
```

Comment:
Does this mean the field always gets populated with a value? For my date pickers I need it to be blank by default as they don't have to fill it in.

Reply:
This is now sorted, very good point. If you want a date setting for an empty field, set default_date to be what you want...